src/TextInput.js

```diff
@@ -162,6 +156,7 @@ class FormidableTextInput extends Component {
           />
         </View>
         {this.props.showError &&
+          this.state.errorMessage &&
```

Comment:
but then, when there's an error, aren't the next inputs pushed down by the error container?